scootapi/scoot.thrift

```diff
@@ -55,7 +55,7 @@ struct TaskDefinition {
 
 struct JobDefinition {
   1: required list<TaskDefinition> tasks,
-  2: optional JobType jobType,
+  2: optional JobType DEPRECATED_jobType,
```

Comment:
will we be removing this? (I'm assuming we're leaving it in to limit the scope of this change)